components/data_comps/datm/bld/build-namelist.pl

```diff
@@ -0,0 +1,415 @@
+#!/usr/bin/env perl
+#-----------------------------------------------------------------------------------------------
+#
+# build-namelist
+#
+# This is the build-namelist script for the CIME datm (Data Atmosphere Model).
+#--------------------------------------------------------------------------------------------
+
```

Comment:
My mistake. It should not be there. I'll delete it. I'm trying to get all of the perl in the data model build-namelists put into one reusable routine that can then be pythonized.